engine-standalone-storage/src/relayer_db/types.rs

```diff
@@ -229,11 +311,25 @@ fn get_address(row: &postgres::Row, field: &str) -> Address {
     Address::try_from_slice(value).unwrap()
 }
 
+struct TransactionDuration(pub u128);
+
+impl TryFrom<TransactionDuration> for u64 {
+    type Error = &'static str;
+
+    fn try_from(value: TransactionDuration) -> Result<Self, Self::Error> {
+        if value.0 > u64::MAX.into() {
+            Err("Value is too large to fit in u64")
+        } else {
+            value.try_into()
+        }
```

Comment:
Why? `u64::try_from` will drop an error if `value.0` won't fit to `64` bits.

Reply:
I could not find a specific error type called `TryFromUnsignedIntError`, so I had to make one.

Reply:
if it was i32 -> {unsigned type}, I would do that, but it seems there is no support for Unsigned int conversion error type.

Reply:
`type Error = TryFromIntError;`
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=02565c9a8436769fcee41c905acf7c7a

Reply:
oh it works, nice.

Reply:
Ah the documentation says the error is for "integral" types, including unsigned int.